apps/web/src/pages/HomePage.tsx

```diff
@@ -1,7 +1,14 @@
+import { useEffect } from 'react';
 import { Navigate } from 'react-router-dom';
 
+const onboarded = localStorage.getItem('onboarded') === 'true';
+
 function HomePage() {
-  return <Navigate to="/templates" replace />;
+  useEffect(() => {
+    localStorage.setItem('onboarded', 'true');
+  }, []);
+
+  return <Navigate to={onboarded ? '/templates' : '/quickstart'} replace />;
```

Comment:
@WinterCore what do you think that instead of approach, we just redirect the user to the `/quickstart` page just after the registration flow. You can check the client code on the register flow, and also on the OAuth flow the server returns an newUser flag. 

Part one Covering web registration:
https://www.loom.com/share/b51796e493334b27a20a153581917046

Part two Covering CLI Registration:
https://www.loom.com/share/e40eaf9444ba4f6e95f9bee324a61cf7
(I pushed this addition of source=cli to your branch)

Let me know if this makes sense :) @WinterCore

Reply:
Hey @scopsy. Thank you so much for the detailed feedback. I completely agree with you, your approach is a lot cleaner.

Question about the `source=cli`, wouldn't this always send users trying to access the dashboard from the CLI to the `/quickstart` page if you decide to add a CLI option to open the dashboard for existing users that is, since it's being handled in the `LoginPage.tsx` component?

Also, do I need to add any cypress tests for this?

Reply:
The existing test you modified actually covers it :) So no need for additional tests. we can merge this.